src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/pages/projects/ProjectUserGroupsPage.java

```diff
@@ -0,0 +1,73 @@
+package ca.corefacility.bioinformatics.irida.ria.integration.pages.projects;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import ca.corefacility.bioinformatics.irida.ria.integration.components.AntTable;
+import ca.corefacility.bioinformatics.irida.ria.integration.pages.AbstractPage;
+
+public class ProjectUserGroupsPage extends AbstractPage {
+	@FindBy(className = "t-add-user-group-btn")
+	private WebElement addUserGroupBtn;
+
+	@FindBy(className = "t-add-user-group-modal")
+	private WebElement addUserGroupModal;
+
+	@FindBy(className = "t-new-member")
+	private List<WebElement> addGroupNameList;
+
+	@FindBy(className = "t-remove-btn")
+	private List<WebElement> removeButtons;
+
+	private static AntTable table;
+
+	public ProjectUserGroupsPage(WebDriver driver) {
+		super(driver);
+	}
+
+	public static ProjectUserGroupsPage goToPage(WebDriver driver, Long projectId) {
+		get(driver, "projects/" + projectId + "/settings/user-groups");
+		table = AntTable.getTable(driver);
+		return PageFactory.initElements(driver, ProjectUserGroupsPage.class);
+	}
+
+	public boolean isAddUserGroupButtonVisible() {
+		return driver.findElements(By.className("t-add-user-group-btn"))
```

Comment:
I think it would be better to return if the size == 1 instead of size > 0. Ensures that there is only one button displayed.

Reply:
Um. Sure.  Done and pushed.